MagicMaster/script.json

```diff
@@ -2,7 +2,7 @@
     "$schema": "https://github.com/DameryDad/roll20-api-scripts/blob/MagicMaster/MagicMaster/Script.json",
     "name": "MagicMaster",
     "script": "MagicMaster.js",
-    "version": "2.047",
+    "version": "2.048",
     "previousversions": ["2.044","2.045","2.046"],
```

Comment:
You would need to add "2.047" to the list of perviousverisions if you want people to still have access to that version.